CipherText/Rotator/Bounds.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jjose00
+ * Date: 11/7/17
+ * Time: 8:22 AM
+ */
+
+namespace Giftcards\Encryption\CipherText\Rotator;
+
+class Bounds implements \Iterator
+{
+    /**
+     * @var int
+     */
+    private $offset;
+    /**
+     * @var null
+     */
+    private $limit;
+    /**
+     * @var int
+     */
+    private $batch;
+
+    /**
+     * @var int
+     */
+    private $key;
+
+    /**
+     * Bounds constructor.
+     * @param int $offset Starting batch number
+     * @param null $limit Max amount of records to process
+     * @param int $batch Amount of records per batch
+     */
+    public function __construct($offset = 0, $limit = NULL, $batch = 1)
```

Comment:
could you lowercase the NULL? Also i think calling it $batchSize will be alot clearer about its purpose